sklearn/ensemble/_forest.py

```diff
@@ -369,6 +369,13 @@ def fit(self, X, y, sample_weight=None):
                     FutureWarning
                 )
 
+            if self.max_features == "auto":
+                warn(
```

Comment:
we should indicate in the warning message what users should do to suppress this warning: either set it to None to get the current behaivour, or set it to 'sqrt' to get the future behaviour.